_socketsGenerator/test/test.test.js

```diff
@@ -22,7 +21,7 @@ describe('Test test.js', function () {
     it('should get a correct parameter', function() {
       result(name);
 
-      assert(result.withArgs(name).called);
+      expect(result.withArgs(name).called).to.be.true;
```

Comment:
What about: 
```
sinon.assert.called(result);
sinon.assert.calledWith(result, name);
```
On my mind you don't check now that method was called with argument if you wanted it.